doc/index.rst

```diff
@@ -32,6 +32,7 @@
     projections/index.rst
     tutorials/coastlines.rst
     tutorials/plot.rst
+    tutorials/line-plot.rst
```

Comment:
I'm wondering if `plot-lines.rst` or `lines.rst` is better.

Reply:
Made the change to `plot-lines.py`.